contrib/utilities/checkdoxygen.py

```diff
@@ -49,6 +49,37 @@ def check_empty_lines_in_tables(lines):
 
     return
 
+# have more than one header with the same name in a tutorial?
+def check_multiple_defined_headers(lines):
```

Comment:
I think you didn't want this commit as part of your patch -- let's leave it for a separate PR.

Reply:
Whoops! I did something funny with my branches and rebasing.

I created this branch at the tip of the script patch branch so I could spot which files had issues but rebasing on master still hangs onto those two commits from the script patch.

Reply:
There we go :wink: